### DISCUSSION TITLE
how is NativeScript-vue different than just using NativeScript?

### DISCUSSION THREAD

**Discussion by fanckush** [nativescript-vue/nativescript-vue#1022] [Q&A] [answered]
in the official NS website, it says that vue is supported 
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/6278713/207721155-cf218d0e-fa1a-4e3a-aabf-7ffda90c7075.png">

So i'm a little confuzed by why ns-vue exists? do I need it to build ns apps with vue js? if not, then what is the benefit?

I saw in the docs the word "plugin" indicating that ns-vue is a plugin for ns, this still doesn't tell me much unfortunately.

**My background**: i'm a frontend developer with vue experience and am looking to build my first ever mobile app, never used NS (and consequently NS-vue) before

**Comment by rigor789** [comment, marked as answer]
NativeScript itself is framework agnostic, the logo on there refers to nativescript-vue, which is technically a "plugin" that makes vue work with the NativeScript provided elements.

**Reply by fanckush** [reply]
thanks!
